postgres/package.json

```diff
@@ -12,14 +12,15 @@
     "name": "Chris Rybicki"
   },
   "peerDependencies": {
-    "winglang": "^0.54.13",
-    "cdktf": "^0.19.1"
-  },
-  "dependencies": {
     "@rybickic/cdktf-provider-neon": "^1.0.0",
-    "@cdktf/provider-aws": "^18.0.4",
-    "@winglibs/containers": "^0.0.22",
-    "pg": "^8.11.3"
+    "@cdktf/provider-aws": "^19.12.0",
+    "@winglibs/containers": "^0.0.26",
+    "pg": "^8.11.3",
+    "cdktf": "^0.20.6"
+  },
+  "overrides": {
+    "@cdktf/provider-aws": "$@cdktf/provider-aws",
+    "cdktf": "$cdktf"
   },
```

Comment:
Temporary fix! `@winglibs/containers` dependency has peer conflicts since it still uses older versions of these deps. A PR will fast follow this one to update any packages that use `@winglibs/containers`